optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -213,6 +216,63 @@ private void applySolverProperties(RecorderContext recorderContext,
                 .collect(Collectors.toList());
     }
 
+    private void stopBuildIfClassesExistThatHavingPlanningAnnotationsButAreNotAPlanningEntity(SolverConfig solverConfig,
+            IndexView indexView) {
+        Collection<AnnotationInstance> optaplannerFieldAnnotations = new HashSet<>();
+
+        for (DotName annotationName : DotNames.PLANNING_ENTITY_FIELD_ANNOTATIONS) {
+            optaplannerFieldAnnotations.addAll(indexView.getAnnotations(annotationName));
+        }
+
+        BiFunction<AnnotationInstance, ClassInfo, String> errorMessage = (annotationInstance, classInfo) -> " with a @" +
+                annotationInstance.name().withoutPackagePrefix() + " annotation is in a class (" + classInfo.name().toString()
+                + ")" +
+                " that does not have a @PlanningEntity annotation. Maybe add a @PlanningEntity annotation to (" +
```

Comment:
Maybe should start on a new line in the output.
```suggestion
                " that does not have a @PlanningEntity annotation.\nMaybe add a @PlanningEntity annotation to (" +
```